source/views/layouts/govuk_template.html.erb

```diff
@@ -110,7 +110,12 @@
             <%= yield :footer_support_links %>
 
             <div class="open-government-licence">
-              <p class="logo"><a href="https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/" rel="license">Open Government Licence</a></p>
+              <% if content_for?(:licence_logo) %>
+                <%= yield :licence_logo %>
```

Comment:
There doesn't seem to be an obvious way to remove the logo without providing an alternative.

How would you use this in your app?